src/clj/rems/workflow/dynamic.clj

```diff
@@ -27,25 +27,120 @@
     :application.state/submitted
     #_:application.state/withdrawn}) ; TODO withdraw support?
 
-(def CommandTypes
-  #{#_:application.command/accept-license
-    #_:application.command/require-license
-    :application.command/accept-invitation
-    :application.command/add-member
-    :application.command/invite-member
-    :application.command/approve
-    :application.command/close
-    :application.command/comment
-    :application.command/decide
-    :application.command/reject
-    :application.command/request-comment
-    :application.command/request-decision
-    :application.command/remove-member
-    :application.command/return
-    :application.command/save-draft
-    :application.command/submit
-    :application.command/uninvite-member
-    #_:application.command/withdraw})
+(s/defschema CommandInternal
+  {:actor UserId
+   ;; TODO: tests use non-DateTime times
+   :time s/Any})
```

Comment:
All the places use the same constant `:time 456`, so it should be quick to search and replace them to use `(DateTime. 1000)` or similar.

Reply:
done